js/plugins/conviva.js

```diff
@@ -283,6 +289,18 @@ var ConvivaAnalyticsPlugin = function(framework)
   };
 
   /**
+   * Validates the custom metadata passed in via page level settings.
+   * @private
+   * @method ConvivaAnalyticsPlugin#validateCustomMetadata
+   * @param  {object} metadata The Conviva custom metadata to validate
+   * @returns {boolean} true if valid, false otherwise
+   */
+  var validateCustomMetadata = function(metadata)
+  {
+    return !!metadata && _.isObject(metadata);
```

Comment:
`!!metadata` seems redundant to me; is there something `!!metadata` will catch that `_.isObject(metadata)` does not?

`_.isObject()` will return `false` for null/undefined